src/UILayer/CQRSStoreGroup.ts

```diff
@@ -14,6 +14,36 @@ import { ChangedPayload } from "../payload/ChangedPayload";
 import { Dispatcher } from "../Dispatcher";
 const CHANGE_STORE_GROUP = "CHANGE_STORE_GROUP";
 
+
+/**
+ * This function is type converter for TypeScript.
+ * It has actual implementation, but it is only use for typing.
+ *
+ * - Status: unstable
+ * 
+ * This is useful for TypeScript, but it is unstable API.
+ * You can use it own risk.
+ * 
+ * ## Example
+ *
+ * ```ts
+ * const mapping = {
+ *   a: new AStore(),
+ *   b: new BStore()
+ * };
+ * const StoreState = MapStoreToState(mapping);
+ * // Get Mapped to State for Store
+ * type States = typeof StoreState
+ * // {
+ * //   a: AState,
+ * //   b: BState
+ * // };
+ * ```
+ */
+export function MapStoreToState<T>(_mapping: MapStore<T>): MapState<T> {
+    return {} as MapState<T>
+}
```

Comment:
@kimamula Great! Thanks! 
I've update this PR.

I have a question. ❓ 
Should we export `MapState` from index.ts?

```ts
// index.ts
export { Dispatcher } from "./Dispatcher";
export { Store } from "./Store";
export { StoreGroup } from "./UILayer/StoreGroup";
// ....
// For TypeScript
export { MapState } from "./UILayer/StoreGroupType"
```

In background, The user can create custom `(CQRS)StoreGroup` class.
Currently, `(CQRS)StoreGroup`-like implementation require `MapState<T>` I think.
But, The user can't get `MapState` interface from public(`index.ts`).